.github/workflows/check-install.yml

```diff
@@ -0,0 +1,52 @@
+name: Check install
+
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency.
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+on:
+  push:
+    paths-ignore:
+      - "doc/**"
+      - "**/README.md"
```

Comment:
So, how about striking ignore-paths README and CHANGELOG?

Reply:
I was thinking the same thing.

Reply:
I went with:

```yaml
    paths-ignore:
      - 'CONTRIBUTING.md'
      - 'doc/**'
      - 'release-notes/**'
      - 'Cabal-tests/test/**'
      - 'cabal-testsuite/PackageTests/**'
 ```

Reply:
On reflection and after reading about [excluding paths](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-excluding-paths), we know the paths that `cabal-install` is dependent on, why don't we switch from `paths-ignore` to `paths`?

```
$ cabal info cabal-install
...
    Dependencies:  ..., Cabal >=3.14 && <3.15,
                   Cabal-syntax >=3.14 && <3.15,
                   cabal-install-solver >=3.14 && <3.15, ...
                   Cabal-QuickCheck, ...
                   Cabal-described, ...Cabal-tests, cabal-install, ...
```

It is also possible to mix, to [exclude and include](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths) paths.

Reply:
Sure